src/main/java/com/google/api/codegen/transformer/ModelTypeFormatter.java

```diff
@@ -26,6 +27,11 @@
  */
 public interface ModelTypeFormatter {
   /**
+   * Get the full name for the given short name, using the default package.
+   */
+  String getFullNameFor(String shortName);
```

Comment:
I think this function should be more specific like the related ones: getImplicitPackageFullNameFor